homeassistant/const.py

```diff
@@ -385,3 +389,4 @@
 TEMPERATURE = 'temperature'  # type: str
 SPEED_MS = 'speed_ms'  # type: str
 ILLUMINANCE = 'illuminance'  # type: str
+BANDWIDTH = 'bandwidth'  # type: str
```

Comment:
I planned to use it, but not on this PR. And I forgot to remove it on the commit. Thanks again